documentation/book/cluster-operator.adoc

```diff
@@ -490,25 +490,30 @@ You can find more information on how to use it in the corresponding GitHub repo.
 
 For more information about using the metrics with Prometheus and Grafana, see <<_metrics>>
 
-===== Logging
-The `logging` field allows the configuration of loggers. These loggers are:
-[source]
-log4j.rootLogger
-connect.root.logger.level
-log4j.logger.org.apache.zookeeper
-log4j.logger.org.I0Itec.zkclient
-log4j.logger.org.reflections
+===== [[logging_examples]]Logging
+The `logging` field allows the configuration of loggers. These loggers for Zookeeper and Kafka are available in <<spec.zookeeper.logging,`spec.zookeeper.logging`>> and <<spec.kafka.logging,`spec.kafka.logging`>> sections above respectively.
```

Comment:
A minor edit... replace " in <<spec.zookeeper.logging,`spec.zookeeper.logging`>> and <<spec.kafka.logging,`spec.kafka.logging`>> sections above respectively."  with " in the <<spec.zookeeper.logging,`spec.zookeeper.logging`>> and <<spec.kafka.logging,`spec.kafka.logging`>> sections  respectively."